### DISCUSSION TITLE
Connection panels - need improvement!

### DISCUSSION THREAD

**Discussion by MaaiG** [FakeFishGames/Barotrauma#7881] [Ideas and suggestions]
Grouping, color notation etc.

**Now:**
![Image 001_17-15](https://user-images.githubusercontent.com/93348499/146780908-ecc611fb-d4ec-4f9e-9015-f9cc2a503c41.png)
**At least this way:**
![143297624-d32b9325-be18-4f4f-ad93-a2a67ab0bfed (1)](https://user-images.githubusercontent.com/93348499/143501249-e6a613f7-60f4-4c1e-8ae4-5f3345e5c431.png)

**Comment by WickedWolfy** [comment]
+1
